### PR TITLE
Adding mikeliguedes's coordinant's

### LINKED ISSUE #15875: Hi!
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by mikeliguedes**
@githubteacher 

Puse mis coordinadas, qué opinas?

closes #15875

### CODE REVIEW

**Comment by mikeliguedes** [comment]
?

File: _pins/mikeliguedes.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: mikeliguedes
```
Comment (cololjp) [inline _pins/mikeliguedes.json]: These file needs the --- at the top --- at the bottom to pass all the tests

**Comment by cololjp** [comment]
You have to add yourself as an Asignee too!

**Comment by Schiavi13** [comment]
@mikeliguedes Necesitas agregar --- al principio y al final

**Pull request merged**